docs/install.md

```diff
@@ -0,0 +1,102 @@
+# Installation
+
+AutoGluon requires Python version 3.7, 3.8, or 3.9. 
+
+::::{dropdown} Optional: Use `conda` to manage Python environment
+You can use [miniconda](https://docs.conda.io/en/latest/miniconda.html) or [anaconda](https://www.anaconda.com/products/distribution) to
+manage your Python environment. Once installed, create an environment with a specific Python version and activate it:
+
+```bash
+conda create -n autogluon python=3.9 -y
+conda activate autogluon
+```
+::::
+
+[pip](https://pip.pypa.io/en/stable/installation/) is the primary way to install AutoGluon, and a recent version is required by AutoGluon's dependencies:
+
+```bash
+python -m pip install --upgrade pip
```

Comment:
I've seen a couple python packages suggest `python -m pip` for the reasons discussed [here](https://stackoverflow.com/questions/60782785/python3-m-pip-install-vs-pip3-install). I did a quick survey of packages in a similar area to us (pandas, numpy, scikit) and they all use simple `pip install`, so I'm fine keeping it simple here and we can change later if we get feedback otherwise.

Reply:
Sounds good, lets go with `pip install` initially as you mentioned. I believe I was the one that switched to using `python -m` at one point early on because of a user getting confused, but it has been several years and I think the likelihood it will cause issues is a lot less than back then.